docker/.env.testnet.example

```diff
@@ -0,0 +1,71 @@
+# File contains docker secific environment varialbes, users should edit only ones in
+# User Configurable section.
+# Each service contains required and optional varialbes, please make sure you populate all required.
+
+#####################
+### User Configurable ###
+
+# Network Env #
+MONIKER=injective
+CHAIN_ID=injective-888
+APP_ENV=staging
+APP_VERSION=v1.7.1
+NETWORK=testnet
+PASSPHRASE=""
+
+# Logging Env #
+LOG_LEVEL="debug"
```

Comment:
Maybe the info is more appropriate; debug can be very noisy.

Reply:
Good catch, my bad. Wasn't able to change it back after I was done debugging.